packages/frontend/app/src/session/Sessions.tsx

```diff
@@ -1,7 +1,6 @@
-import React, { FC, useState } from "react";
+import React, { FC } from "react";
```

Comment:
We should aim to be consistent with how we are importing React and its functions. In `SessionContainer` you deconstructed `UseState` whereas here you took that out and deconstructed `FC` which you took out there! 👀 .